AWSSDKSwiftCLI/Tests/AWSSDKSwiftCLITests/Commands/PackageVersionBuilderTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+@testable import AWSSDKSwiftCLI
+
+class PackageVersionBuilderTests: XCTestCase {
```

Comment:
Tests for the package version builder's generated code, error handling, and default paths.